portfolio/src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+public final class Message {
+    private final long id;
+    private final String comment;
+    private final String yes;
+    private final String email;
+    private final long timestamp;
+
+    /** Add the new message to the list of all messages */
```

Comment:
Is this comment in the wrong place? The associated code is for Message constructor